workspaces/lluis/Image.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="typescript">
+  export let src
+  export let alt
+  export let size
+  export let className
+  export let isRounded = false;
+</script>
+
+<figure class="image { size ? `is-${size}` : '' } {className}" class:is-rounded={isRounded}>
```

Comment:
I would be wary of always putting a figure tag around images. I think this isn't considered valid practice in HTML, as the figure tag has its own meaning.

Before this, `<figure />` was only added when that image was actually illustrating something :thinking: 

I would rather create a separate `<Figure />` component

Reply:
Implemented this in the new commit :)

Reply:
Thanks!